setup.py

```diff
@@ -31,5 +31,5 @@
       packages=find_packages(exclude=['ez_setup', 'examples', 'tests']),
       platforms=['any'],
       zip_safe=False,
-      install_requires=['six'],
+      install_requires=['six', 'defusedxml'],
```

Comment:
I realize this is a fork of a third party library and there is no requirements.txt but my preference is not to use the install_requires.  I think we can do so here given the third party library thing with no alt-way of specifying dependencies, but make sure anywhere use this in a project that we actually call out an pin the version of difusedxml in the requirements.txt

Reply:
Is there a reason why defusedxml is not pinned a version in the setup.py, too?   Will simplexmlapi really work with any future or past version of defusedxml?

Reply:
Yeah I think those are good points about pinning.

Reply:
How about I pin it to >=0.6.0 as that seems to be the latest version?